### DISCUSSION TITLE
Bug: Notification badge still appears when not signed-in to SSO organisation

### DISCUSSION THREAD

**Discussion by Harmelodic** [community/community#17913] [Other Feature Feedback, Questions, & Ideas]
I'm part of an organisation that requires SSO to be able to interact with the organisation and it's repos.

My issue is that when I'm not signed in via SSO, I still see the notification badge on GitHub and I can see the number of notifications that require my attention on the Notifications page.

I would not like to see this since:

1. It is frustrating to see the notification badge telling me I have notifications, but when I access the page there are no notifications, because I'm not signed in.
2. I don't think, for security reasons, I should be able to see how many notifications I have in my inbox when I'm not signed in via SSO to be able to read and deal with them.
3. I do not want to, or am able to, sign in via SSO on every machine that I use to interact with GitHub, just so I can clear the notification badge.

Screenshot attached:

<img width="821" alt="Screenshot 2022-06-04 at 17 36 49" src="https://user-images.githubusercontent.com/9994373/172013524-4b31246a-565e-4095-b8d7-d9de136796c3.png">

**Comment by carloscuesta** [comment]
Duplicate of https://github.com/community/community/discussions/15047
